geometry-schema.json

```diff
@@ -1,91 +1,224 @@
 {
-    "$schema": "http://json-schema.org/schema#",
-    "$id": "http://json-schema.org/geojson/geometry.json#",
-    "title": "geometry",
-    "description": "One geometry as defined by GeoJSON. Licensed as per original source is https://github.com/fge/sample-json-schemas/blob/master/geojson/geometry.json",
-    "type": "object",
-    "required": [ "type", "coordinates" ],
-    "oneOf": [
-        {
-            "title": "Point",
-            "properties": {
-                "type": { "enum": [ "Point" ] },
-                "coordinates": { "$ref": "#/definitions/position" }
-            }
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "http://geojson.org/schema/Geometry.json",
+  "title": "GeoJSON Geometry",
+  "oneOf": [
+    {
+      "$id": "#Point",
```

Comment:
internal $id is not used in the official schema, the rather provide a specific document per type to point directly to it.

Reply:
yes, that's why I've opened https://github.com/geojson/schema/issues/10

Reply:
i like having $id, what I don't like is not hacking the GeoJSON schema ad hoc.

Reply:
but that' something needed, if the GeoJSON Repo people do not respond we cannot be blocked forever ...